README.md

```diff
@@ -36,6 +36,25 @@ app.post('/upload', upload.array('photos', 3), function(req, res, next) {
 })
 ```
 
+### Setting ACL
+
+[ACL values](http://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPOST.html#d0e24378) can be set by passing an optional `acl` parameter into the `multerS3` object.
+
+```javascript
+var upload = multer({
+  storage: multerS3({
+    s3: s3,
+    bucket: 'some-bucket',
+    acl: 'public-read',
+    key: function (req, file, cb) {
+      cb(null, Date.now().toString())
+    }
+  })
+})
+```
+
+Options: `private | public-read | public-read-write | aws-exec-read | authenticated-read | bucket-owner-read | bucket-owner-full-control`.
```

Comment:
Maybe make this a list, and also add in a description of each option?

    Options:

    - `private` - Can only be accessed with valid credentials
    - `public-read` - Can be seen by anyone that has the link
    ...

We can probably copy the description from amazons own documentation somewhere...